cloudinary_test.go

```diff
@@ -0,0 +1,419 @@
+			},
+		},
+		{
+			name: "PutFile_Image",
+			setup: func() {
+				fileInfo := &File{path: "logo.png"}
+				path, err := driver.PutFile("PutFile1", fileInfo)
+				assert.Nil(t, err)
+				assert.True(t, driver.Exists(path))
+				assert.Nil(t, driver.DeleteDirectory("PutFile1"))
+			},
+		},
+		{
+			name: "PutFile_Text",
+			setup: func() {
+				fileInfo := &File{path: "test.txt"}
+				path, err := driver.PutFile("PutFile", fileInfo)
+				assert.Nil(t, err)
+				assert.True(t, driver.Exists(path))
+				data, err := driver.Get(path)
+				assert.Nil(t, err)
+				assert.Equal(t, "Goravel", data)
+				assert.Nil(t, driver.DeleteDirectory("PutFile"))
+			},
+		},
+		{
+			name: "PutFileAs_Text",
+			setup: func() {
+				fileInfo := &File{path: "test.txt"}
+				path, err := driver.PutFileAs("PutFileAs", fileInfo, "text")
+				assert.Nil(t, err)
+				assert.Equal(t, "PutFileAs/text.txt", path)
+				assert.True(t, driver.Exists(path))
+				data, err := driver.Get(path)
+				assert.Nil(t, err)
+				assert.Equal(t, "Goravel", data)
+
+				path, err = driver.PutFileAs("PutFileAs", fileInfo, "text1.txt")
+				assert.Nil(t, err)
+				assert.Equal(t, "PutFileAs/text1.txt", path)
+				assert.True(t, driver.Exists(path))
+				data, err = driver.Get(path)
+				assert.Nil(t, err)
+				assert.Equal(t, "Goravel", data)
+
+				assert.Nil(t, driver.DeleteDirectory("PutFileAs"))
+			},
+		},
+		{
+			name: "PutFileAs_Image",
+			setup: func() {
+				fileInfo := &File{path: "logo.png"}
+				path, err := driver.PutFileAs("PutFileAs1", fileInfo, "image")
+				assert.Nil(t, err)
+				assert.Equal(t, "PutFileAs1/image", path)
+				assert.True(t, driver.Exists(path))
+
+				path, err = driver.PutFileAs("PutFileAs1", fileInfo, "image1.png")
+				assert.Nil(t, err)
+				assert.Equal(t, "PutFileAs1/image1", path)
+				assert.True(t, driver.Exists(path))
+
+				assert.Nil(t, driver.DeleteDirectory("PutFileAs1"))
+			},
+		},
+		{
+			name: "Size",
+			setup: func() {
+				assert.Nil(t, driver.Put("Size/1.txt", "Goravel"))
+				assert.True(t, driver.Exists("Size/1.txt"))
+				length, err := driver.Size("Size/1.txt")
+				assert.Nil(t, err)
+				assert.Equal(t, int64(7), length)
+				assert.Nil(t, driver.DeleteDirectory("Size"))
+			},
+		},
+		{
+			name: "TemporaryUrl",
+			setup: func() {
+				url, err := driver.TemporaryUrl("TemporaryUrl/1.txt", time.Now().Add(5*time.Second))
+				assert.NotNil(t, err)
+				assert.Empty(t, url)
+			},
+		},
+		//{
+		//	name: "Url",
+		//	setup: func() {
+		//		assert.Nil(t, driver.Put("Url/1.txt", "Goravel"))
+		//		assert.True(t, driver.Exists("Url/1.txt"))
+		//		url := url + "/Url/1.txt"
+		//		assert.Equal(t, url, driver.Url("Url/1.txt"))
+		//		resp, err := http.Get(url)
+		//		assert.Nil(t, err)
+		//		content, err := io.ReadAll(resp.Body)
+		//		assert.Nil(t, resp.Body.Close())
+		//		assert.Nil(t, err)
+		//		assert.Equal(t, "Goravel", string(content))
+		//		assert.Nil(t, driver.DeleteDirectory("Url"))
+		//	},
+		//},
```

Comment:
Q: Why remove this?

Reply:
It's temporary. It will have different url every time, so it was hard to find (it wasn't predictable I will replace it later.)

Reply:
For example for the URL?